File: app/src/main/java/com/github/kongpf8848/animation/activity/tween/TweenRotateActivity.java
package com.github.kongpf8848.animation.activity.tween;

import android.content.Intent;
import android.view.View;

import com.github.kongpf8848.xsdk.ui.activity.BaseActivity;
import com.github.kongpf8848.animation.R;
import com.github.kongpf8848.animation.activity.tween.rotate.Rotate_8_Activity;

import butterknife.OnClick;


/**
 * Created by pengf on 2016/12/29.
 * 旋转动画
 */

public class TweenRotateActivity extends BaseActivity
{


    @Override
    protected int getLayoutId() {
        return R.layout.activity_tween_rotate;
    }


    @OnClick(R.id.button1)
    public void onButton1(View view) //图片8旋转动画
    {
        Intent intent=new Intent(this, Rotate_8_Activity.class);
        startActivity(intent);

    }


}

File: app/src/main/java/com/github/kongpf8848/animation/activity/tween/TweenRotateActivity.kt
package com.github.kongpf8848.animation.activity.tween

import android.view.View
import butterknife.OnClick
import com.github.kongpf8848.animation.R
import com.github.kongpf8848.animation.base.BaseToolbarActivity
import com.github.kongpf8848.animation.activity.tween.rotate.MusicActivity
import com.github.kongpf8848.animation.dialog.RingProgressDialog
import com.github.kongpf8848.animation.dialog.ShareDialog

class TweenRotateActivity : BaseToolbarActivity() {

    override fun getLayoutId(): Int {
        return R.layout.activity_tween_rotate
    }

    @OnClick(R.id.button1)
    fun onButton1(view: View?)
    {
        startActivity(MusicActivity::class.java)
    }

    @OnClick(R.id.button2)
    fun onButton2(view: View?)
    {
        RingProgressDialog(this).show("RingProgressDialog")
    }

    @OnClick(R.id.button3)
    fun onButton3(view: View?)
    {
        ShareDialog(this).show("ShareDialog")
    }
}